frontend/capstone-frontend/src/hooks/useAppointments.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react'
+
+export function useAppointments() {
```

Comment:
Tiny tiny nit: Why call this `useAppointments` rather than fetch/get appointments